cpp/test/DataStorm/types/Writer.cpp

```diff
@@ -4,7 +4,8 @@
 
 #include "DataStorm/DataStorm.h"
 #include "Test.h"
-#include "TestHelper.h"
+
+#include <iostream>
```

Comment:
The fact that TestHelper was not used highlights a problem. The tests should use the helper runner. I would fix it in a separate PR.